site/content/docs/5.2/components/toasts.md

```diff
@@ -27,7 +27,7 @@ To encourage extensible and predictable toasts, we recommend a header and body.
 
 Toasts are as flexible as you need and have very little required markup. At a minimum, we require a single element to contain your "toasted" content and strongly encourage a dismiss button.
 
-{{< example class="bg-light" >}}
```

Comment:
I'd say that removing the `class="bg-light"` in this file can be exported directly as a separate PR.